### PR TITLE
Feature/stage frequency

### PR DESCRIPTION
**Pull request by Brennan1994**
Incorporating Stage frequency into the graphical stuff

### CODE REVIEW

**Comment by Brennan1994** [comment]
@codymccoy hoping you can take a look at the incorporation into the compute and verify that looks reasonable to you (the last commit)

File: ViewModel/FrequencyRelationships/GraphicalVM.cs [hunk 1]
```diff
@@ -13,9 +13,12 @@ namespace HEC.FDA.ViewModel.FrequencyRelationships
     public class GraphicalVM : ComputeComponentVM
     {
         private int _equivalentRecordLength = 5;
+        private bool _useStage;
+        private bool _useFlow = true;
         private NamedAction _confidenceLimits;
-        public NamedAction ConfidenceLimits { get { return _confidenceLimits; } set { _confidenceLimits = value; NotifyPropertyChanged(); } }
 
+
+        public NamedAction ConfidenceLimits { get { return _confidenceLimits; } set { _confidenceLimits = value; NotifyPropertyChanged(); } }
```
Comment (codymccoy) [inline ViewModel/FrequencyRelationships/GraphicalVM.cs:21]: It is best to put getters and setters on different lines. This becomes helpful when needing to put breakpoints on them.

File: ViewModel/TableWithPlot/ComputeComponentVM.cs [hunk 2]
```diff
@@ -55,6 +55,7 @@ public string XLabel
             {
                 _xlabel = value;
                 NotifyPropertyChanged();
+                NotifyPropertyChanged(nameof(Units));
```
Comment (codymccoy) [inline ViewModel/TableWithPlot/ComputeComponentVM.cs:58]: Why add another notify prop changed here?

File: ViewModel/TableWithPlot/ComputeComponentVM.cs [hunk 3]
```diff
@@ -67,6 +68,7 @@ public string YLabel
             {
                 _ylabel = value;
                 NotifyPropertyChanged();
+                NotifyPropertyChanged(nameof(Units));
```
Comment (codymccoy) [inline ViewModel/TableWithPlot/ComputeComponentVM.cs:71]: and here?

**Review by codymccoy — COMMENTED** [review]
A couple little things

File: ViewModel/TableWithPlot/ComputeComponentVM.cs (hunk 2 above)
Comment (Brennan1994) [reply ViewModel/TableWithPlot/ComputeComponentVM.cs:58]: The label on the UI is bound to the property "Units" which takes XLabel and YLabel and formats them into a nice string to display. I update Y label when I switch between stage and discharge, so i need units to change when y label changes

File: ViewModel/TableWithPlot/ComputeComponentVM.cs (hunk 3 above)
Comment (Brennan1994) [reply ViewModel/TableWithPlot/ComputeComponentVM.cs:71]: same as above

**Pull request merged**